infrastructure/aws/iam.tf

```diff
@@ -79,3 +79,19 @@ data "aws_iam_policy_document" "github-runner" {
     ]
   }
 }
+
+resource "aws_iam_role" "scheduler_role" {
```

Comment:
Does this execution role need a policy attaching to give it permission to invoke the lambda function?